docs/guide/extending/extending_udw.md

```diff
@@ -49,8 +49,9 @@ There you can set the following properties:
 |active_sort_order</br>`activeSortOrder`|ascending</br>descending|no|Sorting order of the children in the Content Tree.|
 |active_tab</br>`activeTab`|browse</br>search</br>bookmarks|no|Starting tab in the UDW.|
 |active_view</br>`activeView`|finder</br>grid|no|Starting view in the UDW.|
-|allow_content_edit</br>`allowContentEdit`|true</br>false|yes|Visibility of `Edit` content button in UDW.|
+|allow_redirects</br>`allowRedirects`|true</br>false|yes|Allows UDW to redirect to other page, e.g. from Content Structure to Browse in Content Edit.|
 |selected_locations</br>`selectedLocations`|[]</br>[locationId]|no|Location that will be selected automatically.|
+|allow_confirmation</br>`allowConfirmation`|true</br>false|yes|Shows the confirmations buttons in the UDW, if set to false, it will not be possible to confirm selection.|
```

Comment:
```suggestion
|allow_confirmation</br>`allowConfirmation`|true</br>false|yes|Shows confirmations buttons in the UDW. If set to false, it will not be possible to confirm selection.|
```